snap/snapcraft.yaml

```diff
@@ -816,25 +846,44 @@ parts:
 
   kuiper:
     source: https://github.com/emqx/kuiper.git
-    source-tag: 0.4.1
+    source-tag: 1.2.1-beta.1
     plugin: make
     after: [go-build-helper]
     override-build: |
+      export PATH="$SNAPCRAFT_STAGE/go1.15/bin:$GOPATH/bin:$PATH"
```

Comment:
I don't think is required anymore (plus it's wrong as we don't include go 1.15 in the snap).

Reply:
removed